lib/converters/auth_token_converter.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:survey/api/http/response/auth_token_response.dart';
+import 'package:survey/models/auth_token.dart';
+
+extension AuthTokenConverter on AuthTokenResponse {
```

Comment:
I think we can reduce 1 class by adding this `extension` function into `auth_token.dart` like our DM project 😄